zerver/tests/test_users.py

```diff
@@ -3252,6 +3258,55 @@ def test_get_inaccessible_user_ids_when_personal_recipient_is_none(self) -> None
         inaccessible_user_ids = get_inaccessible_user_ids([bot.id, hamlet.id, othello.id], polonius)
         self.assertEqual(inaccessible_user_ids, {othello.id})
 
+    @override_settings(PREFER_DIRECT_MESSAGE_GROUP=False)
+    def test_check_can_access_user_with_personal_message_history(self) -> None:
```

Comment:
After enabling `PREFER_DIRECT_MESSAGE_GROUP`, I had to add some new tests with this setting disabled to reach to 100% test coverage:

![telegram-cloud-photo-size-4-5882193969889265509-x](https://github.com/user-attachments/assets/00e5484f-1ed5-4106-86c8-6abe4253c126)